src/python/pants/option/options_bootstrapper.py

```diff
@@ -110,12 +108,14 @@ def create(
         flags = set()
         short_flags = set()
 
-        # TODO: This codepath probably shouldn't be using FileContent, which is a very v2 engine thing.
+        # We can't use pants.engine.fs.FileContent here because it would cause a circular dep.
+        @dataclass(frozen=True)
+        class FileContent:
+            path: str
+            content: bytes
+
         def filecontent_for(path: str) -> FileContent:
-            is_executable = os.stat(path).st_mode & stat.S_IXUSR == stat.S_IXUSR
```

Comment:
Glad to see this gone. This usage was before we had default arguments for `is_executable`.